src/App.tsx

```diff
@@ -1,14 +1,43 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
+import { useDispatch } from 'react-redux';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+// import { useAppDispatch } from './app/hooks';
```

Comment:
Please consider using the `useAppDispatch` as it's a typed version of the `useDispatch`.